Vagrantfile

```diff
@@ -76,6 +76,11 @@ if $mergePort.nil?
   $mergePort = 8096
 end
 
+$nodeExportPort = ENV['NODE_EXPORT_PORT']
```

Comment:
I had been using a VagrantfileLocal to forward the export server's port in the past. This will make it happen by default.